.changes/next-release/Feature-20c6488b-1750-4ca6-9eb3-b509f49f194b.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Emit additional metadata in telemetry"
```

Comment:
please read https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

Reply:
Do you mean changing the description to something like "Code Transform Telemetry includes additional metadata"? Or are you referring to the part "run `npm run newChange` for each change", and that we should include additional lines like the below?
```
{
	"type": "Feature",
	"description": "Code Transform Telemetry includes additional metadata"
},
{
	"type": "Fix",
	"description": "Code Transform Telemetry missing result metadata"
}
```

Reply:
Updated change log message